### DISCUSSION TITLE
Cannot install datasets (timeout error)

### DISCUSSION THREAD

**Discussion by damlitos** [huggingface/datasets#5417] [General]
Hi 
I wanted to install datasets as below and get an timeout error as indicated below

Code:
from datasets import load_dataset_builder
ds_builder = load_dataset_builder("rotten_tomatoes")

Error:
TypeError: dataset_info() got an unexpTypeError: dataset_info() got an unexpected keyword argument 'timeout'ected keyword argument 'timeout'

what might be the reason?

Thanks

**Comment by mariosasko** [comment]
`timeout` is a valid parameter of `huggingface_hub.dataset_info`, so something could be off with your installation of `huggingface_hub`. What do you get when you run the following code:
```python
import huggingface_hub
huggingface_hub.dataset_info("rotten_tomatoes", timeout=10)
```
